System/Posix/Files.hsc

```diff
@@ -229,7 +229,7 @@ createLink :: FilePath -> FilePath -> IO ()
 createLink name1 name2 =
   withFilePath name1 $ \s1 ->
   withFilePath name2 $ \s2 ->
-  throwErrnoPathIfMinus1_ "createLink" name1 (c_link s1 s2)
+  throwErrnoIfMinus1_ ("createLink (target: "++name1++") (linkpath: "++name2++")") (c_symlink s1 s2)
```

Comment:
This should not change the function from creating a hard link to creating a symbolic link:
```suggestion
  throwErrnoIfMinus1_ ("createLink (target: "++name1++") (linkpath: "++name2++")") (c_link s1 s2)
```

Reply:
This does not appear to be fixed. Why is it *still* calling `c_symlink` here?